core/src/test/scala/org/apache/spark/AccumulatorSuite.scala

```diff
@@ -171,7 +171,7 @@ class AccumulatorSuite extends SparkFunSuite with Matchers with LocalSparkContex
       d.foreach {
         x => acc.localValue ++= x
       }
-      acc.value should be ( (0 to maxI).toSet)
+      acc.value should be ((0 to maxI).toSet)
```

Comment:
I just noticed that there are many other places where 'PropSpec' style is used `with Matchers`, while I guess spark project is mainly using `FunSuite` sylte